LondonBritishLibrary/orient/BLorient539.xml

```diff
@@ -0,0 +1,324 @@
+                                    <dimensions type="outer" unit="in">
+                                        <height>10.25</height>
+                                        <width>9</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">Mutilated. Several leaves have been torn out between 
+                                <locus target="#33"/> and <locus target="#34"/>. Two leaves have been torn out after
+                                <locus target="#172"/>, only a small piece of the first remaining.</condition>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="21">
+                                    <locus from="2" to="184r"/>
+                                </layout>
+                                <layout columns="2" writtenLines="20 22">
+                              <locus from="184v" to="226"/>
+                        </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1650" notAfter="1749"/>
+                                <desc>Written in a good hand.</desc>
+                            </handNote>
+                            
+                            <handNote script="Ethiopic" xml:id="h2" corresp="#a1">
+                                <date notBefore="1650" />
+                                <persName ref="PRS13079HellaweK" role="scribe"/>
+                            </handNote>
+                        </handDesc>   
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1" corresp="#h2">
+                                        <locus from="224v" to="226v"/>
+                                    <desc type="GuestText"><title type="complete" ref="LIT3052RepCh332"/></desc>
+                                    <q xml:lang="gez"><seg part="I">እምሥርወ፡ ዕሴይ፡ እስመ፡ ሠረፅከ፡ ወእማኅፀና፡ ለሩት። ናስተበፅዓከ፡ ዘልፈ፡ በንባበ፡ ዝንቱ፡ ስብሐት። 
+                                        ኦብፁዕ፡ ንጉሠ፡ እስራኤል፡ ዳዊት። ሶበ፡ ተከሥተ፡ ወተረከበ፡ ባሕርየ፡ ውሣጣይ፡ ሕይወት። እመዝገበ፡ ከርሣ፡ ለማርያም፡ ዘለከ፡ ሕይወት። ሰላም፡ ለዝክረ፡ 
+                                        ስምከ፡ ውስተ፡ ረቀ፡ መዝሙር፡ ዘተዌደደ።
+                                   </seg></q>
+                                 </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#226v"/>
+                                    <desc type="OwnershipNote">A note gave the name of a later owner, for which that of 
+                                        <persName ref="PRS13080Isayeyyas"/> has been substituted.</desc>
```

Comment:
The comment on the owner is difficult to understand.

Reply:
I agree. But there is not more information, so I see nothing we can do here without adding our own interpretation.